profile/main.go

```diff
@@ -70,6 +71,7 @@ type Diff struct {
 	TwitterId   string    `firestore:"twitter_id,omitempty"`
 	InstagramId string    `firestore:"instagram_id,omitempty"`
 	Website     string    `firestore:"website,omitempty"`
+	Dob         string    `firestore:"dob,omitempty"`
```

Comment:
why dob is string wrt dateTime

Reply:
thankyou for pointing it out, I have fixed it now.